Leetcode/1347/Minimum_step_no_to_make_two_strings_anagram.cpp

```diff
@@ -0,0 +1,85 @@
+/*
+  Leetcode Problem: 1347
+  Problem link: (https://leetcode.com/problems/minimum-number-of-steps-to-make-two-strings-anagram/)
+*/
+
+#include<iostream>
+#include<vector>
+
+class Solution {
+public:
+    int minSteps(std::string s, std::string t) {
+        // Declare two vectors as frequency array to count the number of characters of a string
```

Comment:
Instead of using two separate frequency arrays for the characters in the strings s and t, we can achieve the same result by using just one frequency array.

What do you think?

Reply:
Okay let me think about it I will trying to make this happen.